modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -217,6 +217,15 @@ For more information about `highlight_on_focus` see the xref:accessibility.adoc#
 [NOTE]
 Any editors using this `highlight_on_focus: true` option, can remove this option from their {productname} init configuration when upgrading to {productname} 7.0.
 
+=== Make table ghost element better reflect height changes when resizing.
+//# TINY-10658
+
+Previously, the height styles applied to the `tr` and `td` elements within the table did not accurately reflect changes in table height when resized using the corner resize handles.
+
+In {productname} 7.0, this problem has been addressed. Now, the height styles have been eliminated from the last row of the table ghost element, allowing it to adjust its height appropriately during resizing.
+
+As a result, the table ghost element now accurately mirrors adjustments in table height when resized using the corner resize handles.
```

Comment:
I feel like mentioning specific details about what changes were made to the ghost element which does not stay in the content might be overkill for the customer to know now that I think about it. Would something like this be ok?
```suggestion
Previously, the table ghost element did not accurately reflect changes in table height when resized using the corner resize handles.

In {productname} 7.0, this problem has been addressed.

As a result, the table ghost element now accurately mirrors adjustments in table height when resized using the corner resize handles.
```